src/core/transformScriptSetup.ts

```diff
@@ -55,6 +56,7 @@ export function transformScriptSetup(
     })
 
   const components = Array.from(template.components)
+    .filter(component => !parserOptions.isNativeTag!(component.toLowerCase()))
```

Comment:
Why should it be `component.toLowerCase()`?

Reply:
Thank you brother, I have exported the original unprocessed `tags `for judgment